packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -401,12 +423,12 @@ const VirtualListBaseFactory = (type) => {
 
 			if (isForward) {
 				// See if the next item is spottable then delegate scroll to onFocus handler
-				if (currentIndex < dataSize - 1 && !data[currentIndex + 1].disabled) {
+				if (currentIndex < dataSize - 1 && !isDisabledItem(currentIndex + 1)) {
```

Comment:
Is there a guard against `isDisabledItem` being undefined here and the cases below?

Reply:
The guard to check if `isDisabledItem` is undefined is the following line that you commented above. So `isDisabledItem` is always not undefined.
https://github.com/enactjs/enact/pull/1524/files/6da91f4e11b5d41c934a6696d5ee2d8dfd26d83d#diff-b6579b4fbd9d50a0d236c66680c0570dR406

Reply:
I see now.  Might revisit this in relation to the question at the comment above.